src/common/meta/src/ddl/drop_table.rs

```diff
@@ -194,15 +189,42 @@ impl Procedure for DropTableProcedure {
 
         LockKey::new(lock_key)
     }
+
+    fn rollback_supported(&self) -> bool {
+        !matches!(self.data.state, DropTableState::Prepare)
```

Comment:
Does physical table need to support rollback? If the data is deleted in the datanode, rolling back metadata is of no use.

Reply:
Yes, the dropping physical table procedure supports rollback. However, dropping the physical table requires dropping the logical tables first(We must restore the metadata). So, we would like to introduce the rollback for the dropping table procedure.